Gemfile

```diff
@@ -17,6 +17,12 @@ group(:maintenance) do
   gem "netrc"
 end
 
+group(:pry) do
+  gem "pry"
+  gem "pry-byebug"
+  gem "pry-stack_explorer"
+end
```

Comment:
I had some questions about whether excluding this group from omnibus builds would have the effect of removing pry from the package, but I learned that it will not.